package.json

```diff
@@ -5,6 +5,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "font-awesome": "^4.7.0",
```

Comment:
Isn't fontawesome 5 available? I don't really like the idea of having an already "old" dependency, but that's fine anyway.

Reply:
Done